src/domain/accounts.ts

```diff
@@ -0,0 +1,181 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Sridhar Voruganti <sridhar.voruganti@modusbox.com>
+
+ --------------
+ ******/
+
+import { Util as HapiUtil } from '@hapi/hapi'
+import Logger from '@mojaloop/central-services-logger'
+
+import {
+  Enum,
+  Util,
+  FspEndpointTypesEnum,
+  RestMethodsEnum
+} from '@mojaloop/central-services-shared'
+// eslint is complaining about these imports. not sure why.
```

Comment:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md

looks like the linter is unable to resolve `~/shared/config` path,but  tsconfig.js setup is ok

Which linter is complaining, the vs code one plugin, or `npm run lint` ?

For me, this line could fix the problem: https://github.com/mojaloop/thirdparty-scheme-adapter/blob/master/.eslintrc.js#L40

there is no such line in thidparty-api-adapter: https://github.com/mojaloop/thirdparty-api-adapter/blob/master/.eslintrc.js

Reply:
Ok fixed now